scripts/feeHandler.js

```diff
@@ -0,0 +1,220 @@
+    return (reserveFeeToBurn.toString() >= KNC_MINIMAL_TX_AMOUNT)
+}
+
+async function enoughWalletFeesToBurn(reserveAddress, walletAddress)
+{
+    let walletFeeToSend = (await FeeBurner.methods.reserveFeeToWallet(reserveAddress, walletAddress).call()).toLowerCase();
+    console.log("walletFeeToSend", KNCWeiToKNCString(walletFeeToSend))
+    return (walletFeeToSend.toString() >= KNC_MINIMAL_TX_AMOUNT)
+}
+
+async function burnReservesFees(reserveAddress) {
+    console.log("burnReservesFees")
+    let enough = await enoughReserveFeesToBurn(reserveAddress);
+    console.log("enough", enough)
+    if (enough) {
+        await sendTx(FeeBurner.methods.burnReserveFees(reserveAddress));
+    }
+}
+
+async function sendFeesToWallets(reserveAddress) {
+    console.log("sendFeesToWallets")
+    for (let wallet in wallets) {
+        let walletAddress = wallets[wallet];
+        console.log("walletAddress", walletAddress)
+        let enough = await enoughWalletFeesToBurn(reserveAddress, walletAddress);
+        console.log("enough", enough)
+        if (enough) {
+            await sendTx(FeeBurner.methods.sendFeeToWallet(walletAddress, reserveAddress));
+        }
+    }
+}
+
+async function reserveKNCWalletDetails(reserveAddress) {
+    let reserveKNCWallet = await FeeBurner.methods.reserveKNCWallet(reserveAddress).call();
+    console.log("reserveKNCWallet", reserveKNCWallet)
+ 
+    let reserveWalletBalance = await KNCToken.methods.balanceOf(reserveKNCWallet).call();
+    console.log("reserveWalletBalance", KNCWeiToKNCString(reserveWalletBalance));
+
+    let reserveWalletAllowance = await KNCToken.methods.allowance(reserveKNCWallet, feeBurnerAddress).call();
+    console.log("reserveWalletAllowance", KNCWeiToKNCString(reserveWalletAllowance));
+}
+
+async function main() {
+
+    getKNCToken()
+
+    getNetwork()
+
+    get_sender()
+    console.log("sender",sender);
+
+    if (typeof gasPriceGwei != 'undefined') {
+        gasPrice = BigNumber(gasPriceGwei).mul(10 ** 9);
+    }
+    else {
+        gasPrice = await web3.eth.getGasPrice()
+    }
+    console.log("gasPrice", gasPrice.toString())
+
+    nonce = await web3.eth.getTransactionCount(sender);
+    console.log("nonce",nonce);
+    
+    let reserves = await Network.methods.getReserves().call();
+    console.log("reserves",reserves);
+
+    feeBurnerAddress = await Network.methods.feeBurnerContract().call();
+    console.log("feeBurnerAddress",feeBurnerAddress);
+
+    FeeBurner = new web3.eth.Contract(JSON.parse(feeBurnerAbi), feeBurnerAddress);
+
+    for (let reserve_index in reserves) {
+        let reserveAddress = reserves[reserve_index];
+        console.log("reserveAddress", reserveAddress)
+        await reserveKNCWalletDetails(reserveAddress)        
+        await burnReservesFees(reserveAddress);
+        await sendFeesToWallets(reserveAddress);
+    }
+
+    console.log("***** performed " + txs +" txs, " + errors + " failed *****")
```

Comment:
would be nice to report amount of ether spent in the process. current balance. and prediction for how much more runs current balance should suffice. Report it in the email.